### ISSUE TITLE
Multiline comments for class properties

### ISSUE THREAD

**Issue by gyahalom** [sphinx-contrib/matlabdomain#125]
Hi, thanks for this great module!
I saw some comments in the code about possible future support for multiline comments before class properties, e.g.
```matlab
classed MyClass
    properties
        % Very long description
        % of property x
        x
    end
end
```
Is this planned/already exists somehow?

**Comment by joeced** [comment]
Hi. Thanks for the comment!

No, it's not available at the moment. Unfortunately, I don't have much time to work on the project, and can't tell when this feature would be available. Pull request are highly welcome.

**Comment by joeced** [comment]
Was fixed in version 0.13.0.
